Documentation/ContentObjects/Records/Index.rst

```diff
@@ -296,10 +290,8 @@ accessible for the current user.
 Selection with categories
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Taken from the rendering of the "Categorized content elements"
-special menu as defined in "css_styled_content".
-
-.. code-block:: typoscript
+If you would want to display categorized content with a RECORDS obeject
```

Comment:
Small typo:

> obeject
object

The rest looks good, but I am not really the TypoScript expert.

Reply:
fixed